local/snapshot.go

```diff
@@ -422,24 +432,39 @@ func (ln *localNetwork) loadSnapshot(
 }
 
 // Remove network snapshot
-func (ln *localNetwork) RemoveSnapshot(snapshotName string) error {
-	return RemoveSnapshot(ln.snapshotsDir, snapshotName)
+func (ln *localNetwork) RemoveSnapshot(
+	snapshotName string,
+	snapshotPath string,
+) error {
+	return RemoveSnapshot(ln.snapshotsDir, snapshotName, snapshotPath)
 }
 
 // Get network snapshots
 func (ln *localNetwork) GetSnapshotNames() ([]string, error) {
 	return GetSnapshotNames(ln.snapshotsDir)
 }
 
-func getSnapshotDir(snapshotsDir string, snapshotName string) string {
+func getSnapshotDir(
```

Comment:
this is very confusing to me.
the logic behind this func

Reply:
maybe make it simple by making this
```
	if snapshotPath != "" {
		return snapshotPath
	}
	``` a very first statement. followed by
	
	```
	if snapshotsDir == "" {
		snapshotsDir = DefaultSnapshotsDir
	}
	return  filepath.Join(snapshotsDir, **snapshotPrefix+snapshotName)
	```
	too make it much less confusing
	
	pls feel free to ignore this comment

Reply:
making the change